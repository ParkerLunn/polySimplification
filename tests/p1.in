5+6x+x^2

