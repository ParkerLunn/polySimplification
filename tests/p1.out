(((x)^2+(6*x))+5)
x^2 + 6x + 5